### DISCUSSION TITLE
Buffering and SD(?) quality

### DISCUSSION THREAD

**Discussion by ghost** [tgorgdotcom/locast2plex#223] [Q&A]
**My setup is as follows:**
- Raspberry Pi 4
- External 1.8 TB HDD connected to USB 3.0 port

**Internet**:  I have Comcast Business and when I run a speed test on the Pi I get 130Mbps down.  So the speed is good.

I have locast2plex and the Plex Media Server (PMS) installed on the Pi.  My Roku Plex app is connected to the PMS on the Pi.   I can successfully DVR shows that are in the Guide on the "Live TV & DVR" on the Plex app on my Roku.  All is good there.

However, the TV shows that I watch from "Live TV & DVR" on the Plex app on my Roku do not look HD at all!!  And, I get the occasional buffering.

Thoughts?

**Comment by CTJohnK** [comment]
Perhaps the Pi is trying to transcode, check the Plex dashboard to see what it is doing when you see this happening.  Also, are you trying multiple streams on the Pi?
